### ISSUE TITLE
Schedule a task to run x seconds for y seconds

### ISSUE THREAD

**Issue by jdahan91** [fluentscheduler/FluentScheduler#230]
Hi, how can i schedule a task to run x seconds [duration] for y seconds [interval]? thanks.

**Comment by VitorCioletti** [comment]
Hi, 
There is no such thing. To solve your problem, you can create your own logic to execute your job during the `y seconds` and then schedule it to run every `x seconds`.

**Comment by jdahan91** [comment]
Let me rephrase what i'm trying to do. I want to run a task for x seconds [duration] but i want it to be called every y seconds [interval]

For example, if i have duration 5 seconds and interval 1 second, the task will run every 1 second for an entire 5 seconds.

**Comment by VitorCioletti** [comment]
Oh, sorry I got you wrong, but the duration of a job is defined by what you is executing, not by FluentScheduler. This library only sets when your job will be called.

**Comment by jdahan91** [comment]
Are you familiar of a framework that is capable of performing what i'm trying to achieve? would it be possible to add support for my request?

**Comment by VitorCioletti** [comment]
> Are you familiar of a framework that is capable of performing what i'm trying to achieve?

Well, i'm not acquainted to these libraries, but you can see if [Hangfire](https://github.com/HangfireIO/Hangfire) or [Quartz.NET](https://github.com/quartznet/quartznet) provides what you are looking for.

> would it be possible to add support for my request?

I do not think this deserves a new feature because it introduces a problem where your job takes much more to execute than defined in `duration` and/or `interval`, also, I do not see much value on it. However, thanks for the suggestion.

:)

Closing this issue, feel free to reopen it.
